src/components/CreateUserModal.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-buttons slot="start">
+          <ion-button @click="closeModal">
+            <ion-icon :icon="closeOutline" slot="icon-only" />
+          </ion-button>
+        </ion-buttons>
+        <ion-title>{{ $t("Create user") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button fill="clear">{{ $t("Save") }}</ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <ion-item lines="full">
+        <ion-label position="fixed">{{ $t("First name") }}</ion-label>
+        <ion-input type="text" />
+      </ion-item>
+      <ion-item lines="full">
+        <ion-label position="fixed">{{ $t("Last name") }}</ion-label>
+        <ion-input type="text" />
+      </ion-item>
+      <ion-item lines="full">
+        <ion-label position="fixed">{{ $t("Email") }}</ion-label>
+        <ion-input type="email" />
+      </ion-item>
+      <ion-item lines="full">
+        <ion-label position="fixed">{{ $t("Phone") }}</ion-label>
+        <ion-input />
+      </ion-item>
+
+      <ion-item lines="none">
```

Comment:
are there no lines in Figma?

Reply:
We need lines here. I have removed this lines none property.